docs/resources/samples/overview.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 'Office Scripts samples'
+description: 'Available Office Scripts samples and solutions.'
+ms.date: 03/18/2021
+localization_priority: Normal
+---
+
+# Office Scripts samples
```

Comment:
Which PR will integrate the existing samples into this?

Reply:
Tracking with [this task](https://office.visualstudio.com/OC/_workitems/edit/4989453) but will try to include in this PR.